assets/styles/stylesl-16.css

```diff
@@ -0,0 +1,144 @@
+html, body {
+    width: 100vw; 
+    height: 100vh; 
+    background-color: rgb(30, 103, 170);
+}
+.grid-container {
+    display: grid;
+    width: 100vw;
+    height: 100vh;
+    grid-template-columns: 100px 1fr 150px;
+    grid-template-rows: 90px repeat(3, 1fr) 90px;
```

Comment:
your container has just 3 rows. 
90px 1fr 90px; is enough

Reply:
Also avoid fixed height. Better set auto 1fr auto. In this case content of the header and footer will define their height.